src/k8s/cmd/k8s/k8s_get_join_token.go

```diff
@@ -52,5 +60,7 @@ func newGetJoinTokenCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
 
 	cmd.Flags().BoolVar(&opts.worker, "worker", false, "generate a join token for a worker node")
 	cmd.Flags().DurationVar(&opts.timeout, "timeout", 90*time.Second, "the max time to wait for the command to execute")
+	// The CLI uses verbose names for flags instead of abbreviations. Internally and for the API, the common TTL (time-to-live) name is used.
+	cmd.Flags().DurationVar(&opts.ttl, "expires-in", time.Hour, "the time until the token expires")
```

Comment:
The API defines the default as 24h but here it is defined as 1 hour? Can we make these consistent?

Reply:
agree here it should be 24h.  That next machine doesn't always start at the speed you want it to.  Factor in installing the snap, going through a proxy -- darn, the token expired...  I like 24h here

Reply:
fair point - updated